src/EntityFramework6.Npgsql/SqlGenerators/SqlBaseGenerator.cs

```diff
@@ -81,7 +82,9 @@ internal Version Version
             {"operator_tsquery_and",Operator.QueryAnd},
             {"operator_tsquery_or",Operator.QueryOr},
             {"operator_tsquery_contains",Operator.QueryContains},
-            {"operator_tsquery_is_contained",Operator.QueryIsContained}
+            {"operator_tsquery_is_contained",Operator.QueryIsContained},
+            {"operator_regex_match",Operator.RegexMatch},
```

Comment:
Please rename ```operator_regex_match_case``` to ```operator_regex_match``` (and remove the other one). This implementation should be only using 1 operator and that's the case sensitive one (and the behavior is then overriden with modifiers according to RegexOptions).